docs/en/operations/system_tables.md

```diff
@@ -139,29 +139,70 @@ This system table is used for implementing the `SHOW DATABASES` query.
 
 ## system.detached_parts {#system_tables-detached_parts}
 
-Contains information about detached parts of [MergeTree](table_engines/mergetree.md) tables. The `reason` column specifies why the part was detached. For user-detached parts, the reason is empty. Such parts can be attached with [ALTER TABLE ATTACH PARTITION|PART](../query_language/query_language/alter/#alter_attach-partition) command. For the description of other columns, see [system.parts](#system_tables-parts). If part name is invalid, values of some columns may be `NULL`. Such parts can be deleted with [ALTER TABLE DROP DETACHED PART](../query_language/query_language/alter/#alter_drop-detached).
+Contains information about detached parts of [MergeTree](table_engines/mergetree.md) tables. The `reason` column specifies why the part was detached. For user-detached parts, the reason is empty. Such parts can be attached with [ALTER TABLE ATTACH PARTITION|PART](../query_language/alter.md#alter_attach-partition) command. For the description of other columns, see [system.parts](#system_tables-parts). If part name is invalid, values of some columns may be `NULL`. Such parts can be deleted with [ALTER TABLE DROP DETACHED PART](../query_language/alter.md#alter_drop-detached).
 
 ## system.dictionaries
 
-Contains information about external dictionaries.
+Contains information about [external dictionaries](../query_language/dicts/external_dicts.md).
 
 Columns:
 
-- `name` (String) — Dictionary name.
-- `type` (String) — Dictionary type: Flat, Hashed, Cache.
+- `database` (String) — Database name where the dictionary is located. Only for DDL dictionaries, for others is always an empty string.
+- `name` (String) — [Dictionary name](../query_language/dicts/external_dicts_dict.md).
+- `status` (Enum8) — Vocabulary status. Possible values: 
+     - `NOT_LOADED` — Dictionary did not load because it was not used.
+     - `LOADED` — Dictionary loaded successfully.
+     - `FAILED` — Unable to load the dictionary as a result of an error.
+     - `LOADING` — Dictionary is loading now.
+     - `LOADED_AND_RELOADING` — Dictionary is loaded successfully, and is being reloaded right now (frequent reasons: SYSTEM RELOADDICTIONARY request, timeout, dictionary config has changed).
```

Comment:
SYSTEM RELOAD DICTIONARY - https://clickhouse.tech/docs/en/query_language/system/#query_language-system-reload-dictionary. Query not request.